documentation/modules/ROOT/pages/configuration/topic-routing.adoc

```diff
@@ -72,7 +72,7 @@ transforms.Reroute.topic.replacement=$1customers_all_shards
 
 `topic.regex`:: Specifies a regular expression that the transformation applies to each change event record to determine if it should be routed to a particular topic.  
 +
-In the example, the regular expression, `(.*)customers_shard(.*)` matches records for changes to tables whose names include the `customers_shard` string. This would re-route records for tables with the following names:
+In the example, the regular expression, `(.* )customers_shard(.*)` matches records for changes to tables whose names include the `customers_shard` string. This would re-route records for tables with the following names:
```

Comment:
```suggestion
In the example, the regular expression, `pass:[(.*)customers_shard(.*)]` matches records for changes to tables whose names include the `customers_shard` string. This would re-route records for tables with the following names:
```
I would actually suggest using the `pass:[]` Asciidoc operator instead so that the text gets passed as-is.  This way we don't give the wrong impression on the regex for those who might actually read the raw adoc or cause issues during a cut-n-paste from the published website.